### PR TITLE
docs(upgrade guide): add config options to deprecation list

### PR DESCRIPTION
**Pull request by tomlongridge**
Added the deprecated/renamed configuration options and linked to online docs.

### CODE REVIEW

**Review by rich-bugsnag — APPROVED** [review]
Thanks!

**Comment by saqibmegaparticle** [comment]
Bugsnag not working with 2019.4.34f1 LTS

File: UPGRADING.md [hunk 1]
```diff
@@ -89,20 +89,33 @@ See our [online docs](https://docs.bugsnag.com/platforms/unity/customizing-bread
...
-| `Bugsnag.SetAutoDetectErrors(bool)`                                | Replaced by setting the config value `AutoDetectErrors` or `EnabledErrorTypes` in configuration or using an OnSendError callback|
-| `Bugsnag.SetAutoDetectAnrs(bool)`                                  | Replaced by setting the config values `AutoDetectErrors` or `EnabledErrorTypes` in configuration or using an OnSendError callback|
+| `Bugsnag.SetAutoNotify(bool)`                                      | Replaced by setting the config value [`AutoDetectErrors`](https://docs.bugsnag.com/platforms/unity/configuration-options/#auto-detect-errors) or [`EnabledErrorTypes`](https://docs.bugsnag.com/platforms/unity/configuration-options/#enabled-breadcrumb-types) in configuration or using an OnSendError callback|
+| `Bugsnag.SetAutoDetectErrors(bool)`                                | Replaced by setting the config value [`AutoDetectErrors`](https://docs.bugsnag.com/platforms/unity/configuration-options/#auto-detect-errors) or [`EnabledErrorTypes`](https://docs.bugsnag.com/platforms/unity/configuration-options/#enabled-breadcrumb-types) in configuration or using an OnSendError callback|
```
Comment (gareththackeray) [inline UPGRADING.md:101]:
```suggestion
| `Bugsnag.SetAutoDetectErrors(bool)`                                | Replaced by setting the config values [`AutoDetectErrors`](https://docs.bugsnag.com/platforms/unity/configuration-options/#auto-detect-errors) or [`EnabledErrorTypes`](https://docs.bugsnag.com/platforms/unity/configuration-options/#enabled-breadcrumb-types) in configuration or using an OnSendError callback|
```

File: UPGRADING.md [hunk 2]
```diff
@@ -89,20 +89,33 @@ See our [online docs](https://docs.bugsnag.com/platforms/unity/customizing-bread
...
 | `Bugsnag.Metadata`                                                 | Replaced by Add/Clear/Get operations – see above.                 |
-| `Bugsnag.SetAutoNotify(bool)`                                      | Replaced by setting the config value `AutoDetectErrors` or `EnabledErrorTypes` in configuration or using an OnSendError callback|
-| `Bugsnag.SetAutoDetectErrors(bool)`                                | Replaced by setting the config value `AutoDetectErrors` or `EnabledErrorTypes` in configuration or using an OnSendError callback|
-| `Bugsnag.SetAutoDetectAnrs(bool)`                                  | Replaced by setting the config values `AutoDetectErrors` or `EnabledErrorTypes` in configuration or using an OnSendError callback|
+| `Bugsnag.SetAutoNotify(bool)`                                      | Replaced by setting the config value [`AutoDetectErrors`](https://docs.bugsnag.com/platforms/unity/configuration-options/#auto-detect-errors) or [`EnabledErrorTypes`](https://docs.bugsnag.com/platforms/unity/configuration-options/#enabled-breadcrumb-types) in configuration or using an OnSendError callback|
```
Comment (gareththackeray) [inline UPGRADING.md:100]:
```suggestion
| `Bugsnag.SetAutoNotify(bool)`                                      | Replaced by setting the config values [`AutoDetectErrors`](https://docs.bugsnag.com/platforms/unity/configuration-options/#auto-detect-errors) or [`EnabledErrorTypes`](https://docs.bugsnag.com/platforms/unity/configuration-options/#enabled-breadcrumb-types) in configuration or using an OnSendError callback|
```

**Review by rich-bugsnag — APPROVED** [review]

**Pull request merged**